examples/reminders/lib/base/common_blocs/coordinator_bloc.dart

```diff
@@ -66,82 +65,82 @@ class CoordinatorBloc extends $CoordinatorBloc {
-  }) =>
-          states.onReminderCreated
-              .whereSuccess()
-              .withLatestFromIdentifiableList(
-                reminderList,
-                operationCallback: operationCallback,
-              )
-              .map((managedList) => ManagedListCounterOperation<ReminderModel>(
-                  managedList: managedList,
-                  counterOperation: CounterOperation.create));
-
   /// Merge the following events with the value of the given [subject]
   /// and emits the result as a new event.
   ///
   /// Supported events:
   /// - reminder update [CoordinatorStates.onReminderUpdated]
   /// - reminder delete [CoordinatorStates.onReminderDeleted]
+  /// - reminder create [CoordinatorStates.onReminderCreated]
+  ///
+  /// Based on the result of the callback [onUpdateOperation] the updated task
+  /// will/will not be removed from value of the provided [subject].
   ///
   /// Based on the result of the callback [onDeleteOperation] the deleted task
-  ///  will be deleted from the value of the provided [subject].
+  /// will be deleted from the value of the provided [subject].
   ///
-  /// Based on the result of the callback [onUpdateOperation] the updated task
-  ///  will/will not be removed from value of the provided [subject].
+  /// Based on the result of the callback [onCreateOperation] the newly create task
+  /// will/will not be merged into the value of the provided [subject].
+
   Stream<ManagedListCounterOperation<ReminderModel>>
       mapReminderManageEventsWithLatestFrom(
     Stream<List<ReminderModel>> reminderList, {
-    required Future<ManageOperation> Function(
-            IdentifiablePair<ReminderModel> model,
+    required Future<ManageOperation> Function(Identifiable model,
             [List<ReminderModel>? list])
         operationCallback,
   }) =>
           Rx.merge([
-            states.onReminderDeleted
+            states.onReminderCreated
                 .whereSuccess()
                 .withLatestFromIdentifiableList(
                   reminderList,
-                  operationCallback: (IdentifiablePair<ReminderModel> reminder,
-                          [List<ReminderModel>? list]) async =>
-                      ManageOperation.remove,
+                  operationCallback: operationCallback,
                 )
                 .map((managedList) =>
                     ManagedListCounterOperation<ReminderModel>(
                         managedList: managedList,
-                        counterOperation: CounterOperation.delete)),
-            states.onReminderUpdated
+                        counterOperation: CounterOperation.create)),
+            states.onReminderDeleted
                 .whereSuccess()
-                .withLatestFromIdentifiablePairList(
+                .withLatestFromIdentifiableList(
                   reminderList,
-                  operationCallback: operationCallback,
+                  operationCallback: (Identifiable reminder,
```

Comment:
The API of OperationCallback is:
`typedef OperationCallback<E extends Identifiable> = Future<ManageOperation> Function(E updatedIdentifiable, List<E> list);`
The `list` parameter is not optional anymore.
Please check all places where you are defining `OperationCallback`.

Reply:
I have changed the `list` parameter not to be optional